conda_forge_tick/utils.py

```diff
@@ -667,6 +715,42 @@ def _parse_validated_recipes(
     return _remove_none_values(parsed_recipes)
 
 
+def parse_recipe_yaml_requirements(requirements):
```

Comment:
I'd suggest using an `_` at the start of the function name here in order to not create more APIs we have ot think about. This one is clearly supposed to be private.

Reply:
Good point!

Done in https://github.com/regro/cf-scripts/pull/2878/commits/2716239b004f3bb74b0b8aa4d72c658932283eba